src/Test.UnitTests.Sarif.Converters/Test.UnitTests.Sarif.Converters.csproj

```diff
@@ -13,6 +13,12 @@
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
 
+  <ItemGroup>
+    <None Remove="TestData\FortifyFprConverter\ExpectedOutputs\SdkScan.sarif" />
```

Comment:
>SdkScan [](start = 63, length = 7)

aah, i need to rename this file to a more appropriate name e.g. scanWithSeverityInfo etc.